### PR TITLE
Feature doc content #5

### PR DESCRIPTION
**Pull request by khkyler**

### CODE REVIEW

File: src/client/app/modules/docs/docs-list.jade [hunk 1]
```diff
@@ -1,4 +1,4 @@
-div.container-fluid.doc-box
+div.sidebar
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/docs-list.jade]: wrap in other class to unique identify docs section

File: src/client/app/modules/docs/docs.jade [hunk 2]
```diff
@@ -1,4 +1,4 @@
 div.row
-  div.doc-list(ui-view="doc-list")
-  div.current-doc(ui-view="container")
+  div.doc-list.sidebar(ui-view="doc-list")
+  div.current-doc.app-content(ui-view="container")
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/docs.jade]: same here. Also, don't create redundant class titles if possible.. If your `ui-view` has a class `app-content`, your containing jade files should not have `app-content` at the top.

File: src/client/app/modules/docs/pages/add.jade [hunk 3]
```diff
@@ -1,11 +1,17 @@
-div.container-fluid.doc-box
+div.app-content.doc-box
   h1 Adding an Endpoint
   div
     p Input the name of the route you want to create
     img.doc-img(src="./images/test.jpg")
     p Click on the + button
+    img.doc-img(src="./images/test.jpg")
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/pages/add.jade]: This should be refactored to select img by:

```
.doc-box
  img
    /* stuff here */
```

per discussion

File: src/client/app/modules/docs/pages/business.jade [hunk 4]
```diff
@@ -1,4 +1,16 @@
...
+    p.
+      Mockr provides the option for the user to add business logic to the data that they persist.  
+      
+      When adding the route or editing it the user can specify some business logic that gets applied to the data
+      
+      as it gets posted to the api endpoint.  
+    p.
+      Some examples of code:
+      
+    span.code-snippet If you have a username property on one of the objects
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/pages/business.jade]: Noticing `.code-snippet` here... any reason to not use `<code>`?

File: src/client/app/modules/docs/pages/mock.jade [hunk 5]
```diff
@@ -1,4 +1,38 @@
...
+      
+      creating your own custom codes up if you need that functionality.  Header data is optional and not neccessary
+      
+      to add to endpoint.  The body data is required however and can be any text string.
+
+    img.doc-img(src="./images/test.jpg")
+    
+      
+      
+    ul List of Standard status codes
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/pages/mock.jade]: Wrap the status codes in code

File: src/client/app/modules/docs/pages/persist.jade [hunk 6]
```diff
@@ -1,4 +1,66 @@
...
+    p.
+      Mockr gives the user the option to have an endpoint persist data.  
+      If the user choses this option the endpoint that they create will have no preloaded data associated with it.  
+      However they will be able to POST PUT and DELETE as well as GET data from that endpoint.
+    p.
+      Persitant data will allow users to add and modify data as if they had a fully funtioning server.
+    p.
+      How to work with your endpoint
+    p.
+      -Get all the data: GET http://mockr-hr.herokuapp.com/1/:username/:route
```
Comment (AndrewSouthpaw) [inline src/client/app/modules/docs/pages/persist.jade]: Make sure all the code snippets here will be wrapped in `<code>`

File: src/client/app/modules/docs/current-doc.jade [hunk 7]
```diff
@@ -1,4 +1,4 @@
-div.container-fluid.doc-box 
+div.app-content.doc-box 
   h1 Welcome to Mockr!
   p Here you will find all the documentation to get started using Mockr.  Mockr will help  speed up your develpoment.
```
Comment (ruben1) [inline src/client/app/modules/docs/current-doc.jade]: one space before speed
development

File: src/client/app/modules/docs/current-doc.jade [hunk 8]
```diff
@@ -1,4 +1,4 @@
...
   p To get started you will need a Github account to log in to Mockr.  If you don't have one go here to sign up:
```
Comment (ruben1) [inline src/client/app/modules/docs/current-doc.jade]: To get started, you..
If you don't have one, go..

**Pull request merged**